index.ts

```diff
@@ -1544,3 +1544,14 @@ export class CallResult<T> {
     return (this._value as Wrapped<T>).inner
   }
 }
+
+export function stringToUTF8(string: String): ByteArray {
```

Comment:
It would be When I read "UTF8", I'm expecting this function to return a string again. It's like saying "String to string". 😉 

Should we make this `Byte(sArray)::fromUTF8(string: String)` perhaps?